src/app/components/log-in/log-in.component.html

```diff
@@ -1,73 +1,81 @@
+            <input
+              id="email"
+              type="text"
+              formControlName="email"
+              placeholder="Email"
+              class="bg-transparent border-b-[1px] border-white/60 h-8 w-full pl-2"
+            />
           </div>
-          <input
-            id="email"
-            type="text"
-            formControlName="email"
-            placeholder="Email"
-            class="bg-transparent border-b-[1px] border-grey/60 h-8 w-full pl-2"
-          />
         </div>
-      </div>
-      <div class="flex flex-col gap-1">
-        <div class="flex items-end w-full">
-          <div class="border-b-[1px] border-grey/60 pl-1">
-            <ng-icon
-              name="heroLockClosed"
-              size="22"
-              class="text-grey/60"
-            ></ng-icon>
-          </div>
-          <input
-            id="password"
-            type="{{ showPassword ? 'password' : 'text' }}"
-            formControlName="password"
-            placeholder="Password"
-            class="bg-transparent border-b-[1px] border-grey/60 h-8 w-full pl-2"
-          />
-          <div class="border-b-[1px] border-grey/60 pl-1">
-            <ng-icon
-              name="{{ showPassword ? 'heroEye' : 'heroEyeSlash' }}"
-              size="22"
-              class="text-grey/60"
-              (click)="togglePassword()"
-            ></ng-icon>
+        <div class="flex flex-col gap-1">
+          <div class="flex items-end w-full">
+            <div class="border-b-[1px] border-white/60 pl-1">
+              <ng-icon
+                name="heroLockClosed"
+                size="22"
+                class="text-white/60"
+              ></ng-icon>
+            </div>
+            <input
+              id="password"
+              type="{{ showPassword ? 'password' : 'text' }}"
+              formControlName="password"
+              placeholder="Password"
+              class="bg-transparent border-b-[1px] border-white/60 h-8 w-full pl-2"
+            />
+            <div class="border-b-[1px] border-white/60 pl-1">
+              <ng-icon
+                name="{{ showPassword ? 'heroEye' : 'heroEyeSlash' }}"
+                size="22"
+                class="text-white/60"
+                (click)="togglePassword()"
+              ></ng-icon>
+            </div>
           </div>
         </div>
-      </div>
-      <div class="flex flex-col gap-2">
-        <button
-          type="submit"
-          class="text-white font-bold px-5 py-2 rounded-3xl self-center w-full mt-4"
-          [class.bg-green]="loginForm.valid"
-          [class.bg-gray-400]="!loginForm.valid"
-        >
-          Login
-        </button>
+        <div class="flex flex-col gap-2">
+          <button
+            type="submit"
+            class="text-white font-bold px-5 py-2 rounded-3xl self-center w-full mt-4 lg:mt-10"
+            [class.bg-button-color]="loginForm.valid"
+            [class.bg-gray-400]="!loginForm.valid"
```

Comment:
Rather than just changing the background colour of the button to grey if the loginForm is invalid, you can consider disabling so that it turns grey and its not clickable